scripts/jarvis.js

```diff
@@ -138,12 +159,16 @@ Everything is running 🎊
 
   Docs: http://localhost:4040
     `);
-    } else if (answers.runConfiguration === 'API Only') {
+    } else if (answers.runConfiguration === API_ONLY) {
       shell.exec('npm run nx build @novu/api');
       shell.exec('npm run start:api');
-    } else if (
-      [RUN_CYPRESS_CLI, RUN_CYPRESS_UI, RUN_CYPRESS_COMPONENT_CLI].includes(answers.runWebConfiguration)
```

Comment:
I introduced a bug for running the Cypress component tests only. Fixing it here.